TheModernIlluminati/TheModernIlluminati/Pages/Index.cshtml

```diff
@@ -19,19 +19,19 @@
     </ol>
     <div class="carousel-inner">
         <div class="carousel-item" data-interval="2000">
-            <img src="~/images/7.gif" width=100% height="350">
+            <img src="~/images/7.gif" width=1150 height="350" />
```

Comment:
Several notes:
1) relative positioning is preferred to absolute.  1150 is very device dependent.
2) Attribute values are not quoted, but should be.
3) Styling should be in linked CSS.
4) Image name is not descriptive, and there is no alt text.